Dockerfile

```diff
@@ -9,7 +9,7 @@
 # docker run -v `pwd`:/go/src/github.com/docker/docker --privileged -i -t docker bash
 #
 # # Run the test suite:
-# docker run -e DOCKER_GITCOMMIT=foo --privileged docker hack/make.sh test-unit test-integration test-docker-py
+# docker run -e DOCKER_GITCOMMIT=foo --privileged docker hack/make.sh test-integration test-docker-py
```

Comment:
`test-unit` removed from this documentation block because the unit tests (or their supporting folders or scripts) appear to have been removed somewhere between branches 17.06 and 17.12.